ush/forecast_postdet.sh

```diff
@@ -527,6 +527,9 @@ CICE_postdet() {
   else  # "${RERUN}" == "NO"
     restart_date="${model_start_date_current_cycle}"
     cice_restart_file="${COM_ICE_RESTART_PREV}/${restart_date:0:8}.${restart_date:8:2}0000.cice_model.res.nc"
+    if [[ "${DO_JEDIOCNVAR:-NO}" == "YES" ]]; then
+      cice_restart_file="${COM_ICE_ANALYSIS}/restart/${restart_date:0:8}.${restart_date:8:2}0000.cice_model_anl.res.nc"
```

Comment:
This may require a change to gdas.cd, in which case we can defer to later, but we would prefer additional directories levels not be added to the COM templates.

Reply:
Yes, good catch again @WalterKolczynski-NOAA . This was discussed offline with @aerorahul and I think we wanted the analysis restart under  ```${COM_ICE_ANALYSIS}/restart/```. 
I can :
- issue the companion pr in the gdasapp with the updated path
- remove the restart directory (I'd rather do that!)

Let me know.

Reply:
I would prefer to get rid of the `restart/` in the path, but I don't know what you and @aerorahul discussed, so I'll wait for him to chime in.

Reply:
@guillaumevernieres and @aerorahul discussed this and we agreed on placing the analysis file in `${COM_ICE_ANALYSIS}/`.  It is not really a `restart` since it is an `analysis`!